__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require("../app/app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+
+beforeAll(() => seed(data));
```

Comment:
This should be `beforeEach`, other wise it will only seed once! For get requests, it is ok, but you will face problems when you get to post requests!